akka-docs/src/main/paradox/fault-tolerance.md

```diff
@@ -1,5 +1,17 @@
 # Fault Tolerance
 
+## Dependency
+
+To use Fault Tolerance, you must add the following dependency in your project.
```

Comment:
Same as with dispatchers, it is not like one uses "failure tolerance", it is more like -- this concept relates to actors, so in order to use these make sure to depend on actors

Reply:
+1